public/modules/custom/grants_preview_link/translations/fi.po

```diff
@@ -0,0 +1,16 @@
+# Finnish translation of City of Helsinki Grants Preview Link module
+#
+msgid ""
+msgstr ""
+
+msgid "Important links"
+msgstr "Tärkeitä linkkejä"
+
+msgid "Allowance terms"
+msgstr "Yleiset avustusehdot"
+
+msgid "/en/allowance-terms"
+msgstr "/fi/yleiset-avustusehdot"
```

Comment:
Siis voiks näin tehdä?

Jotenki tuntuu väärältä. Mut eipä siinä mitää jos pelaa :)